postcode_lookup/templates/includes/registration_timetable_col.html

```diff
@@ -0,0 +1,36 @@
+{% if section.context.with_headers %}
+  <{{ section.context.htag_primary }} id="{{toc_id}}">{% trans %}Voter registration{% endtrans %}</{{ section.context.htag_primary }}>
+{% endif %}
+{% if section.context.can_register %}
+    {% if section.context.with_headers %}
+      <{{ section.context.htag_secondary }}>{% trans %}Register to vote{% endtrans %}</{{ section.context.htag_secondary }}>
+    {% endif %}
+    <p>{% trans %}City of London council elections do not use the same electoral register as other elections.{% endtrans %}</p>
+    <p>
+      {% trans %}
+        Both residents and city workers are eligible to vote. There is one register
+        published annually, and the deadline to apply is 30 November each year.
+        The new register comes into force on 16 February each year,
+        and cannot be modified after that date.
+      {% endtrans %}
+    </p>
+    <p>
+      <a href="https://www.speakforthecity.com/" class="o-button o-button--primary o-button--dark o-external-link">
+        {% trans %}Register to vote in the City of London{% endtrans %}
+      </a>
+    </p>
+{% else %}
+    <{{ section.context.htag_secondary }} id="{{toc_id}}">{% trans %}Registration in the City of London{% endtrans %}</{{ section.context.htag_secondary }}>
+    <p>{% trans %}City of London council elections do not use the same electoral register as other elections.{% endtrans %}</p>
+    {%
+        trans
+        date=section.data.date|date_filter,
+        registration_deadline=section.timetable.registration_deadline|date_filter
+    %}
+    <p>The deadline to register to vote in City of London local elections on {{ date }}
+        was {{ registration_deadline }}.</p>
+    <p>If you registered to vote before the deadline, then you will be able to vote in this election. <a
+            href="#electoral_services">Contact City of London</a> to check if you are on the register.</p>
+    <p>You can <a href="https://www.speakforthecity.com/">register to vote in the City of London</a> in future elections.</p>
+    {% endtrans %}
+{% endif %}
```

Comment:
I haven't updated any translations in this PR. See https://github.com/DemocracyClub/ec-postcode-lookup-pages/pull/110

Reply:
Also, all this text is open to change if it needs it. I've mostly used the same text Peter gave me that we used in https://github.com/DemocracyClub/WhoCanIVoteFor/pull/2071